helix-rest/src/test/java/org/apache/helix/rest/server/AbstractTestClass.java

```diff
@@ -564,6 +564,8 @@ private void preSetupForParallelInstancesStoppableTest(String clusterName,
     clusterConfig.setFaultZoneType("helixZoneId");
     clusterConfig.setPersistIntermediateAssignment(true);
     _configAccessor.setClusterConfig(clusterName, clusterConfig);
+    RESTConfig emptyRestConfig = new RESTConfig(clusterName);
```

Comment:
Let's not create fake RESTConfig that does not have the required fields.

If causes unexpected behavior on https://github.com/apache/helix/blob/master/helix-rest/src/main/java/org/apache/helix/rest/clusterMaintenanceService/MaintenanceManagementService.java#L459C9-L459C27